baselines/dom.generated.d.ts

```diff
@@ -7536,11 +7536,12 @@ declare var HashChangeEvent: {
 interface History {
     readonly length: number;
     readonly state: any;
-    back(distance?: any): void;
-    forward(distance?: any): void;
-    go(delta?: any): void;
-    pushState(statedata: any, title?: string, url?: string): void;
-    replaceState(statedata: any, title?: string, url?: string): void;
+    scrollRestoration: ScrollRestoration;
+    back(): void;
+    forward(): void;
+    go(delta?: number): void;
+    pushState(data: any, title: string, url?: string | null): void;
```

Comment:
optional parameter doesn't need the `| null` annotation, it gets that automatically.
Also, if missing the value of the optional parameter should be `undefined` I think